projects/cypress/CY8CKIT_064S0S2_4343W/mtb/jitp/config_files/iot_pkcs11_config.h

```diff
@@ -32,8 +32,16 @@
 #ifndef _AWS_PKCS11_CONFIG_H_
 #define _AWS_PKCS11_CONFIG_H_
 
-/* A non-standard version of C_INITIALIZE should be used by this port. */
-/* #define pkcs11configC_INITIALIZE_ALT */
+/* Logging configuration for the PKCS #11 library. */
+#ifndef LIBRARY_LOG_NAME
```

Comment:
Would be good to have the comment section similar to what [we have on C-SDK](https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/development/demos/mqtt/mqtt_demo_basic_tls/mqtt_config.h#L24-L34) that describes how these macros tie the PKCS library with the logging stack.

You also need to include `logging_levels.h` before the definition of these macros for accessing the `LOG_XX` macros

Reply:
Added this comment. Thanks for calling out `logging_levels.h` missed copying that into most of thes